tests/phpunit/admin/test-class-wc-rest-stripe-orders-controller.php

```diff
@@ -188,4 +188,63 @@ public function test_capture_payment_invalid_status() {
 
 		remove_filter( 'pre_http_request', $test_request, 10, 3 );
 	}
+
+	public function test_capture_payment_refunded_order() {
+		wp_set_current_user( 1 );
+		$order = WC_Helper_Order::create_order();
+		wc_create_refund(
+			[
+				'order_id' => $order->get_id(),
+				'amount'   => 1,
+			]
+		);
+
+		$endpoint = self::ORDERS_REST_BASE . '/' . strval( $order->get_id() ) . '/capture_terminal_payment';
+		$request = new WP_REST_Request( 'POST', $endpoint );
+		$request->set_param( 'payment_intent_id', 'pi_12345' );
+		$response = rest_do_request( $request );
+
+		$this->assertEquals( 400, $response->get_status() );
```

Comment:
@brianyu28 question here, not really related to this PR. This case returns the code "wc_stripe_missing_order", defined [here](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2211/files#diff-06ec44aaeff9c055d096e82b1f2a6b19a818f53d4f089417cbf46d14ddf9ac4eR130). We use that same code when no order exists with the given order ID, defined [a few lines above](https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2211/files#diff-06ec44aaeff9c055d096e82b1f2a6b19a818f53d4f089417cbf46d14ddf9ac4eR125).
Wouldn't it be convenient to have different error codes for these two cases?

And another one more related to this PR. How about testing the returned error codes are the expected ones as well? Thinking that this would prevent changing the codes by mistake at some point, in case the integration with this endpoint might be relying on them.

Reply:
> Wouldn't it be convenient to have different error codes for these two cases?

Good catch, thanks @a-danae, that's a copy/paste error on my part. I've updated the error code in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2215/commits/f7049f9132f564f965f3511b24232442d674b8fb so that the two cases now have different codes.

> How about testing the returned error codes are the expected ones as well?

Makes sense to me, added in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2215/commits/dab66259b1364301d8f280718221ad8d2f04216a.

PHPCS warnings addressed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2215/commits/8cfa26e9133ce0a8d3ac012fdaca647c2b4b6b75.

Reply:
Sweet, no more comments on my end 🙂  Thank you, @brianyu28!